src/bp-templates/bp-nouveau/buddypress/common/js-templates/activity/parts/bp-activity-post-form-avatar.php

```diff
@@ -1,7 +1,7 @@
 <script type="text/html" id="tmpl-activity-post-form-avatar">
-	<# if ( data.display_avatar ) { #>
+	<# if ( data.display_avatar ) {  #>
 	<a class="activity-post-avatar" href="{{data.user_domain}}">
-		<img src="{{data.avatar_url}}" class="avatar user-{{data.user_id}}-avatar avatar-{{data.avatar_width}} photo" width="{{data.avatar_width}}" height="{{data.avatar_width}}" alt="{{data.avatar_alt}}" />
+		<img src="{{data.avatar_url.split("#038;").join("")}}" class="avatar user-{{data.user_id}}-avatar avatar-{{data.avatar_width}} photo" width="{{data.avatar_width}}" height="{{data.avatar_width}}" alt="{{data.avatar_alt}}" />
```

Comment:
```suggestion
		<img src="{{{data.avatar_url}}}" class="avatar user-{{data.user_id}}-avatar avatar-{{data.avatar_width}} photo" width="{{data.avatar_width}}" height="{{data.avatar_width}}" alt="{{data.avatar_alt}}" />
		
		
```

@shohel I don't think we need to split here anything just use the `{{{` insted of `{{` and let me know this will fix the issue or not.

Reply:
@chetansatasiya  It's working, made changes